cltk/prosody/latin/clausulae_analysis.py

```diff
@@ -18,32 +18,38 @@ def __init__(self):
         return
 
     @staticmethod
-    def clausulae_analysis(prosody):
+    def clausulae_analysis(prosody: List) -> Dict:
         """
         Return dictionary in which the key is a type of clausula and the value is its frequency.
         :param prosody: the prosody of a prose text (must be in the format of the scansion produced by the scanner classes.
         :return: dictionary of prosody
+        >>> Clausulae().clausulae_analysis(['-u--x', '-uuuux'])
+        {'cretic trochee': 1, 'cretic trochee resolved a': 0, 'cretic trochee resolved b': 0, 'cretic trochee resolbed c': 0, 'double cretic': 0, 'molossus cretic': 0, 'double/molossus cretic resolved a': 0, 'double/molossus cretic resolved b': 0, 'double/molossus cretic resolved c': 0, 'double/molossus cretic resolved d': 0, 'double/molossus cretic resolved e': 0, 'double/molossus cretic resolved f': 0, 'double/molossus cretic resolved g': 0, 'double/molossus cretic resolved h': 0, 'double trochee': 0, 'double trochee resolved a': 0, 'double trochee resolved b': 0, 'hypodochmiac': 0, 'hypodochmiac resolved a': 0, 'hypodochmiac resolved b': 1, 'spondaic': 0, 'heroic': 0}
         """
 
         prosody = ''.join(prosody)
 
         return {
-            'cretic + trochee': prosody.count('¯˘¯¯x'),
-            '4th paeon + trochee': prosody.count('˘˘˘¯¯x'),
-            '1st paeon + trochee': prosody.count('¯˘˘˘¯x'),
-            'substituted cretic + trochee': prosody.count('˘˘˘˘˘¯x'),
-            '1st paeon + anapest': prosody.count('¯˘˘˘˘˘x'),
-            'double cretic': prosody.count('¯˘¯¯˘x'),
-            '4th paeon + cretic': prosody.count('˘˘˘¯¯˘x'),
-            'molossus + cretic': prosody.count('¯¯¯¯˘x'),
-            'double trochee': prosody.count('¯˘¯x'),
-            'molossus + double trochee': prosody.count('¯¯¯¯˘¯x'),
-            'cretic + double trochee': prosody.count('¯˘¯¯˘¯x'),
-            'dactyl + double trochee': prosody.count('¯˘˘¯˘¯x'),
-            'choriamb + double trochee': prosody.count('¯˘˘¯¯˘¯x'),
-            'cretic + iamb': prosody.count('¯˘¯˘x'),
-            'molossus + iamb': prosody.count('¯¯¯˘x'),
-            'double spondee': prosody.count('¯¯¯x'),
-            'cretic + double spondee': prosody.count('¯˘¯¯¯¯x'),
-            'heroic': prosody.count('¯˘˘¯x')
+            'cretic trochee': prosody.count('-u--x'),
```

Comment:
You've clearly put a lot of thought into the prosody metrics.
Consider using a named tuple or a class.
A dictionary is fine, but it's a bit ad hoc. 
Let's provide structure for users wherever possible.

Reply:
Sounds reasonable, I used a named tuple to model the rhythms which did make the code much cleaner